tracer.go

```diff
@@ -116,7 +114,7 @@ type Tracer interface {
 
 // StartSpanOptions allows Tracer.StartSpan() callers and implementors a
 // mechanism to override the start timestamp, specify Span References, and make
-// Tags available at Span start time.
+// a single Tag or multiple Tags available at Span start time.
```

Comment:
eh, I take it back given the context of this comment... so maybe `... and make one or more Tags available ...`